tests/integration/team_test.py

```diff
@@ -23,7 +23,7 @@
 import pytest
 
 import libnmstate
-from libnmstate.error import NmstateLibnmError
+from libnmstate.error import NmstateDependencyError
```

Comment:
Would be good to mention the new exception type in the commit message and later in the changelog

Reply:
Done, thanks!